tools/ci_build/github/azure-pipelines/templates/jobs/download_win_qnn_sdk.yml

```diff
@@ -0,0 +1,26 @@
+parameters:
+  - name: QnnSDKVersion
+    type: string
+    default: '2.19.2.240210'
+
+steps:
+  - powershell: |
+      azcopy.exe cp --recursive https://lotusscus.blob.core.windows.net/models/qnnsdk/qnn-v${{ parameters.QnnSDKVersion }}_win $(Agent.TempDirectory)
```

Comment:
where is the license file on blobstore?
can we place it at https://lotusscus.blob.core.windows.net/models/qnnsdk/
and then download it to $(QnnSDKRootDir) ?

Reply:
The license file is uploaded to that same qnnsdk directory in azure. I can add a step to this job that copies it to the local $(QnnSDKRootDir).

Reply:
Do we still need to modify cmake to copy the license file from `--qnn_home` into build directory?

Reply:
right. i think it can be done along with the setup.py change for packaging. let's hold off on that for now.